README.md

```diff
@@ -59,6 +61,38 @@ NOTE: `VULKAN_LOADER_GENERATED_DIR` should be the directory that contains `vk_la
 
 See [docker/build.Dockerfile](docker/build.Dockerfile) for detailed Ubuntu build instructions.
 
+## Enabling the layers:
+To enable the performance layers, set these environent variables:
+
+`$LD_LIBRARY_PATH`: Need to append the directory containing the .so files of performance layers to the `$LD_LIBRARY_PATH`.
```

Comment:
Another solution is to copy layers `.so`s an existing directory listed in `LD_LIBRARY_PATH` (and potentially some system lib dirs?)